src/Dfe.Spi.Common/Dfe.Spi.Common.Caching/Managers/MemoryCacheManager{TCacheKey,TManagerItem}.cs

```diff
@@ -0,0 +1,124 @@
+﻿namespace Dfe.Spi.Common.Caching.Managers
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Dfe.Spi.Common.Caching.Definitions.Caches;
+    using Dfe.Spi.Common.Caching.Definitions.Managers;
+    using Dfe.Spi.Common.Logging.Definitions;
+
+    /// <summary>
+    /// Implements <see cref="IMemoryCacheManager{TCacheKey, TManagerItem}" />.
+    /// </summary>
+    /// <typeparam name="TCacheKey">
+    /// The type of key used in the underlying storage.
+    /// </typeparam>
+    /// <typeparam name="TManagerItem">
+    /// The type of item being managed.
+    /// </typeparam>
+    public class MemoryCacheManager<TCacheKey, TManagerItem>
```

Comment:
Again, should not need typed manager as nothing in logic cares (or should) about underlying ICacheProvider implementation

Reply:
Code updated to use strings and objects, as suggested.